core/src/com/unciv/ui/civilopedia/CivilopediaCategories.kt

```diff
@@ -188,6 +188,11 @@ enum class CivilopediaCategories (
         getImage = null,
         KeyCharAndCode('D'),
         "OtherIcons/Tyrannosaurus"
+    ),
+    GameSpeed ("Game Speeds", false,
+        getImage = null,
+        KeyCharAndCode('S'),
+        "EmojiIcons/Turn"
```

Comment:
One of the textures going in at 50x50, the rest is bigger. Probably just about visible. Your choice if that displeases you. Can always be changed later (actually all the category icons should be independently moddable and not reuse names used elsewhere, the packer will merge identical copies as long as they're in the same atlas pack - me too I was too lazy often for new uses)

Reply:
I see now that now. I've gotten a new (attributed, royalty-free) icon uploaded for use as the game speed civilopedia icon and anywhere else that someone sees fit.

Reply:
Sorry for being too terse before. If you looked up the hourglass in credits.md and re-downloaded that as base, likely fine, maybe mention the new use. Any other Art must be attributed in said credits.md file, _with_ license type. Your new entry is likely within some "unless otherwise stated everything is Creative Commons xyz" scope, so please check whether your source has a clear license policy and whether that matches. All licenses must be compatible with our own license, which I forget what it is. All very rigid thinking, annoying in a way, but a project like this needs to abide by the powers.

Reply:
Oh, and not your fault, but mine - the auto-reformatting the editor did is probably wrong. Try to copy the markdown into any github markdown preview and compare... If it renders the same, fine, if not, try to revert. And tell me. Could go into `.editorconfig` if I find the proper keywords.